config/config.go

```diff
@@ -53,9 +53,7 @@ var (
 type Config struct {
 	models.Host
 	PrivateKey        wgtypes.Key                     `json:"privatekey" yaml:"privatekey"`
-	MacAddress        net.HardwareAddr                `json:"macaddress" yaml:"macaddress"`
```

Comment:
Removing these will probably need NC GUI updates in the settings page

Reply:
these were duplicate attributes; they are already in modes.Host